catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswQueryFactoryTest.java

```diff
@@ -254,19 +253,17 @@ public void setUp()
 
     queryFactory = new CswQueryFactory(cswRecordMap, filterBuilder, filterAdapter);
 
-    AttributeRegistry mockAttributeRegistry = mock(AttributeRegistry.class);
-    when(mockAttributeRegistry.lookup(TITLE_TEST_ATTRIBUTE))
-        .thenReturn(Optional.of(mock(AttributeDescriptor.class)));
-    queryFactory.setAttributeRegistry(mockAttributeRegistry);
+    AttributeRegistryImpl attributeRegistry = new AttributeRegistryImpl();
+    attributeRegistry.registerMetacardType(getCswMetacardType());
```

Comment:
Why is this getCswMetacardType() method still there? I thought you removed the CswMetacardType

Reply:
These unit tests are testing against those particular attributes in the `CswMetacardType`. Initially, I was going to mock the `AttributeRegistry`, but it was going to be a pain since I would need when statements for many attributes. I thought it made more sense to make a real `AttributeRegistry` and initialize it with the `CswMetacardType` that this test is already setting up and asserting against.

Reply:
This test probably shouldn't be making up its own version of the `CswMetacardType` to assert against, but I thought that was out of the scope of this PR. Honestly I think we could have a whole epic for cleaning up the CSW code because it's a mess in a lot of ways.

Reply:
I agree with this.  We've added new dynamics like the QueryFilterTransformer that will render a bunch of code obsolete.  Ideally we would move GMD to that dynamic anyway.

Reply:
Ok. Might be worth leaving a comment or renaming that method, even though it's just in the tests, to not say CswMetacardType because that's confusing since there isn't a CswMetacardType at all.